### ISSUE TITLE
can not find "fc_conv_padding" parameter in function vgg.vgg_16

### ISSUE THREAD

**Issue by ghost** [warmspringwinds/tf-image-segmentation#22]
when i read the code fcn_32s.py , i find that it called function vgg.vgg_16 and passing into a parameter called fc_conv_padding="SAME".  
            logits, end_points = vgg.vgg_16(mean_centered_image_batch,
                                            num_classes=number_of_classes,
                                            is_training=is_training,
                                            spatial_squeeze=False,
                                            fc_conv_padding='SAME')

However, when I turn to the vgg.vgg_16, i can not find this parameter.

def vgg_16(inputs,
           num_classes=1000,
           is_training=True,
           dropout_keep_prob=0.5,
           spatial_squeeze=True,
           scope='vgg_16'):

**Comment by ghost** [comment]
you can actually remove `fc_conv_padding='SAME'` because, in definition of  `vgg.vgg_16,` the argument for `slim.conv2d` is already included ( `padding='VALID'`)
